pkg/server/chaosd/network.go

```diff
@@ -296,3 +303,67 @@ func (s *Server) recoverDNSServer(attack *core.NetworkCommand) error {
 
 	return nil
 }
+
+func (s *Server) applyPortOccupied(attack *core.NetworkCommand) error {
+
+	if len(attack.Port) == 0 {
+		return nil
+	}
+
+	flag, err := checkPortIsListened(attack.Port)
+	if err != nil {
+		if flag {
+			return errors.Errorf("port %s has been occupied", attack.Port)
+		} else {
+			return errors.WithStack(err)
+		}
+	}
+
+	if flag {
+		return errors.Errorf("port %s has been occupied", attack.Port)
+	}
+
+	c := fmt.Sprintf("nc -l %s", attack.Port)
+	cmd := exec.Command("sh", "-c", c)
+	err1 := cmd.Start()
+	if err1 != nil {
+		return errors.WithStack(err1)
+	}
+	return nil
+}
+
+func checkPortIsListened(port string) (bool, error) {
+	checkStatement := fmt.Sprintf("lsof -i:%s | awk '{print $2}' | grep -v PID", port)
+	cmd := exec.Command("sh", "-c", checkStatement)
+
+	stdout, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Error(cmd.String()+string(stdout), zap.Error(err))
+		if err.Error() == "exit status 1" && string(stdout) == "" {
+			return false, nil
+		} else {
+			return true, errors.WithStack(err)
+		}
+	}
+
+	if string(stdout) == "" {
+		return false, nil
+	} else {
+		return true, nil
+	}
+}
+
+func (s *Server) recoverPortOccupied(attack *core.NetworkCommand, uid string) error {
+
+	c := fmt.Sprintf("lsof -i:%s | awk '{print $2}' | grep -v PID | xargs kill -9", attack.Port)
+
+	cmd := exec.Command("sh", "-c", c)
+
+	stdout, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Error(cmd.String()+string(stdout), zap.Error(err))
```

Comment:
how about adding some description for the message, for example:
```suggestion
		log.Error(fmt.Sprintf("execute command: %s, output: %s", cmd.String(), string(stdout)), zap.Error(err))
```